src/modes/summary/Summary.svelte

```diff
@@ -69,18 +68,15 @@
   </RegionDatePicker>
   <div class="uk-container content-grid">
     <div class="grid-3-11">
-      <FancyHeader invert>{region.displayName}</FancyHeader>
-      <Overview {date} {region} {fetcher} />
```

Comment:
Removing the `Overview` element also means we lose the nested element for [`IndicatorWarning`](https://github.com/cmu-delphi/www-covidcast/blob/f38810486ba0365a55c30370d37169d083e2b973/src/blocks/IndicatorWarning.svelte#L32-L42), which provides helpful information in the case that there is a data lapse.  Can we re-add that?